docs/source/starters/new_project_tools.md

```diff
@@ -1,4 +1,250 @@
-# Configure a new project
+# Tools to customise a new Kedro project
```

Comment:
```suggestion
# Tools to configure a new Kedro project
```

IMO the tools are for configuring rather than customising, because it doesn't allow users to pick their own linter or data folder structure etc.

Reply:
The main problem I had with "configure" is that it's a general term (although so is "customise"!) and this documentation sits alongside the section already marked "Configuration" which is configuration of quite a different type! I felt like it was confusing to use the same terminology for a different context, and think "customise" is a relatively general term for something that allows you to pick and choose different options.

Reply:
Made changes to use `customise` consistently in [eb41d2d](https://github.com/kedro-org/kedro/pull/3331/commits/eb41d2defb1c7b38bbe631cc07a09a0f87f1c4cd)